tests/jobserver/test_github.py

```diff
@@ -112,16 +111,18 @@ def test_get_repos_with_branches():
 
 
 @responses.activate
-def test_githuborganizationoauth2_user_data_404():
+def test_githuborganizationoauth2_user_data_204():
     expected_url = "https://api.github.com/orgs/opensafely/members/test-username"
-    responses.add(responses.GET, url=expected_url, status=404)
+    responses.add(responses.GET, url=expected_url, status=204)
 
     class DummyBackend(GithubOrganizationOAuth2):
```

Comment:
While you're here, could you pull `DummyBackend` into the module, and document what it's for?

Reply:
Ooh, good idea.  I'll make it a fixture 👍